src/runtime/Makefile

```diff
@@ -384,6 +420,22 @@ ifneq (,$(QEMUCMD))
 
     KERNELSENAME = kata-containers-se.img
     KERNELSEPATH = $(KERNELDIR)/$(KERNELSENAME)
+
+    NVKERNELNAME = $(call MAKE_NVKERNEL_NAME,$(KERNELTYPE))
```

Comment:
Optional observation: It looks like we are already inconsistent with our naming, but the examples we have are either KERNEL(NAME/PATH/...)_<variant> e.g. `KERNELPATH_FC`, `KERNELPATH_CLH`, or KERNEL<variant>(NAME/PATH/...) e.g. `KERNELTDXPATH`, `KERNELSEVPATH`, so whilst we don't have a single standard, it might be helpful to try and match one of them? I think the former is probably the first and easiest to find, so might be my slight preference, but appreciate that adding Nvidia versions of TDX and SNP adds complexity to this.

Reply:
We need to cleanup here ... 
```
   KERNEL_NAME_FC = $(call MAKE_KERNEL_NAME,$(KERNELTYPE_FC))
    KERNELPATH_FC = $(KERNELDIR)/$(KERNEL_NAME_FC)

```
I like this one: 
`KERNELPATH_FC` `KERNELPATH_CLH`

So do something likes this: `KERNELPATH_NV` `KERNELCONFIDENTIALPATH_NV` ?

Reply:
Or `KERNELPATH_NVCONFIDENTIAL` ?

Reply:
Yep - that option works for me. Thanks